demos/python_demos/human_pose_estimation_demo/README.md

```diff
@@ -0,0 +1,110 @@
+the argument, refer to **When to Reverse Input Channels** section of
+[Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+```
+python3 human_pose_estimation.py -h
+```
+The command yields the following usage message:
+```
+usage: human_pose_estimation.py [-h] -m MODEL -i INPUT [-d DEVICE]
+                                [-t PROB_THRESHOLD] [-r]
+                                [-nireq NUM_INFER_REQUESTS]
+                                [-nstreams NUM_STREAMS]
+                                [-nthreads NUM_THREADS] [-loop LOOP]
+                                [-no_show] [-u UTILIZATION_MONITORS]
+                                [--type {ae,openpose}] [--tsize TSIZE]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model.
+  -i INPUT, --input INPUT
+                        Required. Path to an image, video file or a numeric
+                        camera ID.
+  -d DEVICE, --device DEVICE
+                        Optional. Specify the target device to infer on; CPU,
+                        GPU, FPGA, HDDL or MYRIAD is acceptable. The sample
+                        will look for a suitable plugin for device specified.
+                        Default value is CPU.
+  -t PROB_THRESHOLD, --prob_threshold PROB_THRESHOLD
+                        Optional. Probability threshold for poses filtering.
+  -r, --raw_output_message
+                        Optional. Output inference results raw values showing.
+  -nireq NUM_INFER_REQUESTS, --num_infer_requests NUM_INFER_REQUESTS
+                        Optional. Number of infer requests
+  -nstreams NUM_STREAMS, --num_streams NUM_STREAMS
+                        Optional. Number of streams to use for inference on
+                        the CPU or/and GPU in throughput mode (for HETERO and
+                        MULTI device cases use format
+                        <device1>:<nstreams1>,<device2>:<nstreams2> or just
+                        <nstreams>)
+  -nthreads NUM_THREADS, --num_threads NUM_THREADS
+                        Optional. Number of threads to use for inference on
+                        CPU (including HETERO cases)
+  -loop LOOP, --loop LOOP
+                        Optional. Number of times to repeat the input.
+  -no_show, --no_show   Optional. Don't show output
+  -u UTILIZATION_MONITORS, --utilization_monitors UTILIZATION_MONITORS
+                        Optional. List of monitors to show initially.
+  --type {ae,openpose}  Optional. Type of the network, either "ae" for
+                        associative embeddingor "openpose" for openpose.
+  --tsize TSIZE         Optional. Target input size. By default target size is
```

Comment:
the phrase below seems badly formed. what do you actually mean? the model reshaped to match input image size?

Reply:
This is a tricky part. Different networks use different resize regimes by default. OpenPose net (0001) resizes the image to a predefined *width*. AE-like nets (0002/0003/0004) resize image to a predefined size of *short side*. After that *networks are reshaped* to fit the input image. So you should specify the target size of the *image* somehow. By default, input shape from IR is used to derive it (please note, that networks will likely be inferred with a different input resolution), but it can be specified manually by the user via this CLI argument.

Reply:
@druzhkov-paul may be just put these words into description?

Reply:
Done.